app/src/main/java/ru/vssemikoz/newsfeed/adapters/NewsFeedAdapter.java

```diff
@@ -60,8 +62,9 @@ public NewsViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType
         final TextView author;
         final ImageButton changeFavoriteStateButton;
         final ProgressBar progressBar;
+        final ImageButton shareNewsButton;
 
-        NewsViewHolder(View view, OnRecyclerItemClickListener listener) {//жду базовый
+        NewsViewHolder(View view, OnRecyclerItemClickListener listener) {
             super(view);
             OnNewsItemClickListener finalListener = (OnNewsItemClickListener) listener;
```

Comment:
`finalListener` is still an ugly name for this guy. Let's choose a better one like `onNewsItemClickListener`.